lib/BootBot.js

```diff
@@ -58,13 +58,32 @@ class BootBot extends EventEmitter {
   }
 
   sendGenericTemplate(recipientId, elements, options) {
+    var image_aspect_ratio = options && options.image_aspect_ratio && options.image_aspect_ratio.toLowerCase();
+    image_aspect_ratio = (image_aspect_ratio !== 'horizontal' && image_aspect_ratio !== 'square') ? 'horizontal' : image_aspect_ratio;
```

Comment:
We don't want to force the user to only `horizontal` or `square`. It's probably not very relevant in this case but it is in the case of the `top_element_style` method bellow. Not enforcing it has a few advantages:

1. If Facebook decides to add a new option in the future, we won't have to update this code to support that option.
2. If the user misspells an option or tries to use an invalid one, Facebook will throw an error and we'll give the user the chance to fix it.
3. This param is not required, so we don't even need to specify a default (we'll let Facebook handle it).

I think we can get rid of this line completely.